flowmachine/flowmachine/features/subscriber/displacement.py

```diff
@@ -115,30 +115,31 @@ def __init__(
 
         self.start = start
 
-        if modal_locations:
-            if isinstance(modal_locations, ModalLocation):
-                hl = modal_locations
+        if reference_location:
```

Comment:
Since `reference_location` is now mandatory, there is no need for this if statement any more. I'd also remove the commented else branch below please.